rerun_cpp/src/rerun/archetypes/boxes3d_ext.cpp

```diff
@@ -59,26 +64,37 @@ namespace rerun {
                 half_sizes.emplace_back(size.x() / 2.0, size.y() / 2.0, size.z() / 2.0);
             }
 
+            // Move the vector into a component batch.
             return Boxes3D::from_half_sizes(std::move(half_sizes));
         }
 
         Boxes3D Boxes3D::from_mins_and_sizes(
             const std::vector<datatypes::Vec3D>& mins, const std::vector<datatypes::Vec3D>& sizes
```

Comment:
I wonder if we can use the same trick for non-component arguments like these 🤔

Reply:
Adding a todo pointing to #3794